adapters/clients/replication.go

```diff
@@ -60,15 +61,38 @@ func (c *replicationClient) FetchObject(ctx context.Context, host, index,
 
 func (c *replicationClient) Exists(ctx context.Context, host, index,
 	shard string, id strfmt.UUID,
-) (objects.Replica, error) {
-	resp := objects.Replica{}
-	req, err := newHttpReplicaRequest(ctx, http.MethodGet, host, index, shard, "", id.String(), nil)
+) (bool, error) {
+	path := fmt.Sprintf("/indices/%s/shards/%s/objects/%s", index, shard, id)
+	method := http.MethodGet
+	url := url.URL{Scheme: "http", Host: host, Path: path}
+	q := url.Query()
+	q.Set("check_exists", "true")
+	url.RawQuery = q.Encode()
+
+	req, err := http.NewRequestWithContext(ctx, method, url.String(), nil)
 	if err != nil {
-		return resp, fmt.Errorf("create http request: %w", err)
+		return false, errors.Wrap(err, "open http request")
```

Comment:
Maybe we should keep using `fmt.Errorf` here instead (same for the rest of this file)

Reply:
Sure 
This was a copy and past from remote index.
It is there just not break code in the next PR this method should be removed all together